src/hooks/useAddCasesInputs.tsx

```diff
@@ -53,6 +54,14 @@ const useAddCasesInputs = (
     [facilityModelVersions],
   );
 
+  const updateInputs = useCallback((update: ModelInputsPopulationBrackets) => {
+    setInputs((inputs) => ({ ...inputs, ...update }));
+  }, []);
```

Comment:
`updateInputs` is being exported and passed along to `AgeGroupGrid` as `updateModel`, but following that trail I think we are passing it a different callback than it expects. Would it make sense to remove this prop to `AgeGroupGrid` and  add `const [model, updateModel] = useModel();` in the `AddCasesModal` instead?

Reply:
I'm having a little trouble untangling this but I _think_ it's done this way because we wanted to decouple this form from the `EpidemicModelContext`? otherwise all the changes would immediately affect the surrounding context, which is probably driving UI changes on the page (in a FacilityRow or on the facility page), which is generally not what we want when we're in this modal.

Reply:
Oh right that makes sense, that means if we did use `updateModel` instead of `updateInputs` it would update the chart while changing the inputs on the modal form. Okay, in that case then does it make sense to you to keep this in a `useCallback` since it is also being exported and used in the inputs?

Reply:
I don't see where `updateInputs` getting used, other than inside the effect on L62? which is why I don't think it needs to be a callback or even a separate function at all; you could just call `setInputs` directly inside the effect. unless I am missing something ...?

Reply:
ugh nevermind I didn't expand the whole file, now I see it. forget everything I said and leave it the way you have it! 🤦